backend/core/src/cron/cron.service.ts

```diff
@@ -1,10 +1,45 @@
 import { Injectable } from "@nestjs/common"
 import { Cron, CronExpression } from "@nestjs/schedule"
+import { UserListQueryParams } from "../auth/dto/user-list-query-params"
+import { UserService } from "../auth/services/user.service"
+import { AuthContext } from "../auth/types/auth-context"
+import { Compare } from "../shared/dto/filter.dto"
+import { EmailService } from "../shared/email/email.service"
+import { ListingsService } from "../listings/listings.service"
 
 @Injectable()
 export class CronService {
-  @Cron(CronExpression.EVERY_WEEK)
-  handleCron() {
-    // TODO: add a cron task here.
+  constructor(
+    private readonly emailService: EmailService,
+    private readonly listingsService: ListingsService,
+    private readonly userService: UserService
+  ) {}
+
+  @Cron(CronExpression.EVERY_1ST_DAY_OF_MONTH_AT_NOON)
+  async handleCron() {
+    if (!process.env.SEND_NOTIFICATIONS_FOR_UPDATE_LISTINGS_REMINDER) {
+      return
+    }
+
+    const userQueryParams: UserListQueryParams = {
+      filter: [
+        {
+          $comparison: Compare["EQUALS"],
+          isPartner: true,
```

Comment:
Could we also add filtering on the user being opted in to email notifications so we aren't grabbing any that have opted-out?

Reply:
This happens later in the code since we don't really have a way of joining the two tables easily with the existing api.